pygmt/datasets/earth_age.py

```diff
@@ -5,20 +5,19 @@
 The grids are available in various resolutions.
 """
 
+from collections.abc import Sequence
 from typing import Literal
 
 from pygmt.datasets.load_remote_dataset import _load_remote_dataset
-from pygmt.helpers import kwargs_to_strings
 
 __doctest_skip__ = ["load_earth_age"]
 
 
-@kwargs_to_strings(region="sequence")
```

Comment:
This line can be removed because `region` is passed to `_load_remote_dataset` and in that function `region` can be a sequence.